crates/torii/core/src/sql.rs

```diff
@@ -33,14 +33,25 @@ pub const FELT_DELIMITER: &str = "/";
 #[path = "sql_test.rs"]
 mod test;
 
-#[derive(Debug, Clone)]
+#[derive(Debug)]
 pub struct Sql {
```

Comment:
We should also update the set_event_message and other functions. Like set_model_member. Or dont they need changes?